knowledge-base/actions/github-script/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'GitHub Script'
```

Comment:
I don't think we can do this one using KB. We will need to write custom code for this. You can remove it for the time being.